src/Tutor.Core/UseCases/Management/Stakeholders/StakeholderService.cs

```diff
@@ -1,26 +1,37 @@
 ﻿using FluentResults;
+using System;
+using System.Text.RegularExpressions;
 using Tutor.Core.BuildingBlocks;
 using Tutor.Core.BuildingBlocks.Generics;
 using Tutor.Core.Domain.Stakeholders;
 using Tutor.Core.Domain.Stakeholders.RepositoryInterfaces;
 
 namespace Tutor.Core.UseCases.Management.Stakeholders;
 
-public class StakeholderService<T> : CrudService<T>, IStakeholderService<T> where T : Stakeholder
+public partial class StakeholderService<T> : CrudService<T>, IStakeholderService<T> where T : Stakeholder
```

Comment:
No need to modify the class in this way. I think you can achieve this with the `Regex.Replace`.

However, I think this logic needs to be reworked in general, so we should schedule a quick call to cleanup the domain and user interplay.